vpr/src/draw/draw.cpp

```diff
@@ -719,9 +719,20 @@ bool draw_if_net_highlighted(ClusterNetId inet) {
 void act_on_key_press(ezgl::application* /*app*/, GdkEventKey* /*event*/, char* key_name) {
     //VTR_LOG("Key press %c (%d)\n", key_pressed, keysym);
     std::string key(key_name);
+    if (key == "Escape") {
```

Comment:
Put a comment in of what you're doing.  Hitting Esc means you deselect anything that might be highlighted.